lms/static/js/sysadmin/mgmt_commands.js

```diff
@@ -88,14 +88,17 @@
     }
 
     function submitForm(event){
+
+        var form = $('#command-form');
+        var form_data = form.clone();
+        form_data.find('input:not([required])').filter(function() { return $(this).val() == ''; }).remove();
```

Comment:
ok, rereading this code I think I figured it out.

You're just filtering out the empty non-required fields to make the sent data smaller, correct?

so here is the re-edited code stanza:
```javascript
form_data.find('input:not([required])').filter(function() { 
    return $(this).val() === ''; 
}).remove();
```